browser-test/src/support/admin_predicates.ts

```diff
@@ -8,12 +8,15 @@ export class AdminPredicates {
   }
 
   async addPredicate(questionName: string, action: string, scalar: string, operator: string, value: string) {
+    await this.page.waitForLoadState('load');
     await this.page.click(`text="${questionName}"`);
-    await this.page.selectOption('.cf-predicate-action select', { label: action });
-    await this.page.selectOption('.cf-scalar-select select', { label: scalar });
-    await this.page.selectOption('.cf-operator-select select', { label: operator });
-    await this.page.fill('.cf-predicate-value-input input', value);
-    await this.page.click('text=Submit');
+    await this.page.selectOption('.cf-predicate-action:visible select', { label: action });
+    await this.page.selectOption('.cf-scalar-select:visible select', { label: scalar });
+    await this.page.selectOption('.cf-operator-select:visible select', { label: operator });
+
+    // TODO: determine whether the value is an input or select

```

Comment:
Should this be done in this PR, or is this TODO for a follow-up PR?

Reply:
I'm trying to do it in a follow-up, since this one just uses a text question. I have a follow-up in progress that adds a predicate for each type of scalar/value